src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -51,7 +51,7 @@ const AclInput = ({ value, onChange, disabled, maxAccessLevel, autoFocus, ...pro
           accessLevel: o.value,
           ...Utils.switchCase(o.value,
             ['READER', () => ({ canCompute: false, canShare: false })],
-            ['WRITER', () => ({ canCompute: true, canShare: false })],
+            ['WRITER', () => ({ canCompute: !isAzureWorkspace, canShare: false })],
```

Comment:
Make sure that we are guaranteed to have received the Ajax response for the workspace details before this executes (we don't know something is an Azure workspace until the response is received).